x-pack/plugin/core/template-resources/src/main/resources/watch-history.json

```diff
@@ -1,6 +1,6 @@
 {
   "index_patterns": [ ".watcher-history-${xpack.watcher.template.version}*" ],
-  "priority": 2147483647,
+  "priority": 2147483648,
```

Comment:
Same here

Reply:
Also `2147483647` was `Integer.MAX_VALUE`, right? I'm not sure anything bad happens if we exceed that, but it seems interesting.

Reply:
Sure thing. Reverted.

@joegallo 
> Also 2147483647 was Integer.MAX_VALUE, right? I'm not sure anything bad happens if we exceed that, but it seems interesting.

So playing about with this, the code did seem to happily accept this priority value in a template and correctly store and return it without overflowing or erroring... That said, I would not be supprised if it overflows at some point when doing comparisons to other priorities and silently evaluates this as -2147483646.